common3.json

```diff
@@ -6131,743 +6131,743 @@
-      "text" : "Fire Spin: 対象が火傷状態である"
+      "text" : "Fire Spin: target is burned"
     },
     {
       "id" : 61566,
-      "text" : "Fire Spin: 対象が宿られ状態である"
+      "text" : "Fire Spin: target is seeded"
     },
     {
       "id" : 61567,
-      "text" : "Fire Spin: 対象が混乱状態である"
+      "text" : "Fire Spin: target is confused"
     },
     {
       "id" : 61568,
-      "text" : "Fire Spin: 対象がAttract状態である"
+      "text" : "Fire Spin: target is infatuated"
     },
     {
       "id" : 61569,
-      "text" : "Fire Spin: 対象が呪い状態である"
+      "text" : "Fire Spin: target is cursed"
     },
     {
       "id" : 61570,
-      "text" : "Fire Spin: 対象が滅び状態である"
+      "text" : "Fire Spin: target is under perish song"
     },
     {
       "id" : 61571,
-      "text" : "Fire Spin: 対象が凍り状態である"
+      "text" : "Fire Spin: Target is frozen"
     },
     {
       "id" : 61572,
-      "text" : "Fire Spin: 対象が交代不可能状態である"
+      "text" : "Fire Spin: Target has an unchangeable status"
     },
     {
       "id" : 61573,
-      "text" : "自己解凍炎系　自分が凍り状態である"
+      "text" : "自己解凍炎系　user is frozen"
     },
     {
       "id" : 61574,
-      "text" : "自己解凍炎系　対象が?%の確立で火傷する可能性がある"
+      "text" : "自己解凍炎系　Target has a ?% Probability of being burned"
     },
     {
       "id" : 61575,
-      "text" : "自己解凍炎系　味方が「Helping Hand」を選択した"
+      "text" : "自己解凍炎系　Ally chose Helping Hand"
     },
     {
       "id" : 61576,
-      "text" : "自己解凍炎系　対象が凍り状態である"
+      "text" : "自己解凍炎系　Target is frozen"
     },
     {
       "id" : 61577,
-      "text" : "Blaze Kick: クリティカルが出やすい"
+      "text" : "Blaze Kick: High crit rate"
     },
     {
       "id" : 61578,
-      "text" : "Blaze Kick: 対象が火傷する可能性がある"
+      "text" : "Blaze Kick: Target can be burned"
     },
     {
       "id" : 61579,
-      "text" : "Blaze Kick: 味方が「Helping Hand」を選択した"
+      "text" : "Blaze Kick: Ally chose Helping Hand"
     },
     {
       "id" : 61580,
-      "text" : "Blaze Kick: 対象が凍り状態である"
+      "text" : "Blaze Kick: Target is frozen"
     },
     {
       "id" : 61581,
       "text" : "Freezing: Has chance to freeze target"
     },
     {
       "id" : 61582,
-      "text" : "Freezing: Ally used helping hand"
+      "text" : "Freezing: Ally chose Helping Hand"
     },
     {
       "id" : 61583,
       "text" : "Freezing: Opponent's other pokemon already frozen"
     },
     {
       "id" : 61584,
-      "text" : "Paralysis: 対象が?%の確立で麻痺する可能性がある"
+      "text" : "Paralysis: Target has a ?% Probability of being paralyzed"
     },
     {
       "id" : 61585,
-      "text" : "Paralysis: 味方が「Helping Hand」を選択した"
+      "text" : "Paralysis: Ally chose Helping Hand"
     },
     {
       "id" : 61586,
-      "text" : "Sleep: 対象が?%の確立で眠る可能性がある"
+      "text" : "Sleep: Target has a ?% Probability of falling asleep"
     },
     {
       "id" : 61587,
-      "text" : "Sleep: 対象以外の相手ポケモン（控え含む）が既に眠っている"
+      "text" : "Sleep: Opponent Pokémon other than target（including spares）are already asleep"
```

Comment:
I forgot to search and replace these parentheses, but might as well replace them while you are at it with the possible typo above

Reply:
I don't see any typo here, what do you mean?

Reply:
I'll clarify the translation. By Spare I meant any reserve Pokémon